### ISSUE TITLE
Template broken

### ISSUE THREAD

**Issue by FynnSchapdick** [adospace/reactorui-maui#97]
Hi guys, the current version of the template broke.
`app.SetWindowsSpecificAssetsDirectory("Assets");` this line broke, because the maui guys did a typo i think ^^. Maybe update all dependencies and this issue should be fixed.

**Comment by adospace** [comment]
I made this typo and fixed it in one of the latest releases.
If you update the template to the latest version it should be aligned with the correct name of the method ...Asset.. instead of ...Assect...

**Comment by FynnSchapdick** [comment]
Ok, this would be right, but i dont think u fixed it in the current version 1.0.131, as the template is the one from your Readme file.

**Comment by Code-DJ** [comment]
@adospace the bug is that the csproj file created by the template still uses "1.0.117" instead of the latest. If you update all to 1.0.131 it works:

```
    <PackageReference Include="Reactor.Maui" Version="1.0.117" />
    <PackageReference Include="Reactor.Maui.Canvas" Version="1.0.117" />
    <PackageReference Include="Reactor.Maui.ScaffoldGenerator" Version="1.0.117" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```
